src/main/java/org/broadinstitute/consent/http/db/MetricsDAO.java

```diff
@@ -27,7 +27,7 @@
 
   @SqlQuery(
     "SELECT * from ( "
-      + "SELECT e.*, v.vote finalvote, v.rationale finalrationale, v.createdate finalvotedate, MAX(e.electionid) "
+      + "SELECT e.*, v.vote finalvote, v.rationale finalrationale, v.updatedate finalvotedate, MAX(e.electionid) "
```

Comment:
Looking at `VoteDAO`, the only time we set a create date is on the update method. On dev, there are many more records with null update dates than create dates:
```
consent=> select count(voteid) from vote;
 count 
-------
 16971
(1 row)

consent=> select count(voteid) from vote where createdate is null;
 count 
-------
 14182
(1 row)

consent=> select count(voteid) from vote where updatedate is null;
 count 
-------
 16489
(1 row)
```

In ElectionDAO, we use the createDate for the final vote date exclusively so I don't think this is the correct approach.